ISSUE_TEMPLATE.md

```diff
@@ -1,23 +1,12 @@
-### Context
+<!--
 
-* **mapbox-gl-js version**: 
+To help us diagnose your problem quickly, please
 
-### Report
+ - include a link to a minimal demonstration the bug (we recommend using https://jsbin.com)
```

Comment:
I recommended jsbin over jsfiddle. 

jsfiddle is opinionated about dividing up css, javascript, and html into separate inputs which is a pain when copy-pasting examples and esiting websites.